lib/tictactoe/game.rb

```diff
@@ -15,41 +20,73 @@ def draw
       @board.draw
     end
 
-    def check_position(pos)
-      if @board.positions[pos] != "-"
-        raise "Error"
+    def get_symbol()
+      puts "Enter symbol"
+      while true
+        symbol = @stdin.gets.chomp
+        if !@validation.check_input_symbol(symbol)
+        else return symbol         end
       end
-      return true
     end
 
-    def play(move, symbol)
-      @symbols.push(symbol)
-      if @symbols[-2] and @symbols[-1] and @symbols[-2] == @symbols[-1]
+    def get_position()
+      puts "Enter position"
+      while true
+        position = @stdin.gets.chomp.to_i
+        if !@validation.check_input_position(position)
+        elsif !@validation.check_board_position(position, @board.positions)
+        else return position         end
```

Comment:
Since nothing happens after the `if` and `elsif` statements, I wonder if you can consolidate line 36-38.